moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryCouncilDistricts.js

```diff
@@ -1,13 +1,16 @@
 import React from "react";
 import { Box, Grid, Typography } from "@mui/material";
 
-// reduce the array of geography objects into an array of city council districts
+// reduce the array of project_geography objects into an array of city council districts
 const reduceDistricts = (data) => {
   const initialValue = [];
-  const districts = data.reduce(
-    (acc, component) => [...acc, component["council_districts"]],
-    initialValue
-  );
+  const districts = data.reduce((acc, geography) => {
+    /* Not all components have geography data */
+    const geographyCouncilDistricts = geography
+      ? geography["council_districts"]
```

Comment:
i swear prettier used to handle this: this can be `geography.council_districts`.